### DISCUSSION TITLE
GOPRO 5.3K H265 converted AV1

### DISCUSSION THREAD

**Discussion by Pax709** [gopro/labs#657] [General]
Hello, is it worth already converting the material to AV1 to reduce the archive, we are talking about the original files that are unlikely to be processed anymore. Does anyone know the parameters(ShutterEncoder) that need to be set in order for the AV1 encoder with RTX 4070 to do it losslessly ?, original material 5.3 K bitrate max. 140000kb/s.

**Comment by Pax709** [comment]
I did some tests and there are no miracles, even with a high bitrate in AV1 the shadows are more heavily compressed, at this stage I conclude that it does not make sense. Tested in Davinci Studio alone, shutterencoder gave much worse results.

**Comment by dnewman-gpsw** [comment]
Converted from Issue to discussion (this is not a Labs bug.)

AV1 is not anything super efficient, most codecs lose something without carefully designed multi-pass encoding.  The camera's HEVC is not multi-pass, the power that would use..., this means even HEVC re-encoding of camera source is possible (with up to 2-3X bitrate savings.)  However the RTX encoder, just like the camera encoder, is designed for speed, not compression efficiency.  You would likely need to spend days experimenting with FFMPEG settings to get good quality and efficient encoding.  FFMPEG is not as good as what is technically possible. This is very specialized, and very hard to do well, NetFlix is pretty good at it delivering good 4K 24p in 15Mb/s, yet I still prefer 80-100Mb/s 24p off a Kaleidescape media server (fancy and overkill.)  GoPro's own cloud service, doesn't try to 'optimized' your source, as there is always risk some quality will be lost.

**Reply by Pax709** [reply]
Thank you David for your answer, that is, H265 can still be treated with two passes for the highest quality and to reduce the file?, what bitrate then to set when the source has a max of 140Mb/s? 

I was doing compression with one H265 pass from the original 140Mb/s file to 100Mb/s and , you can see a very slight decrease in detail in the shadows and it is perfectly acceptable.

**Reply by dnewman-gpsw** [reply]
At least two passes, with a quality, not a bitrate target, VBR is better for archive.

**Reply by Pax709** [reply]
I have read a little bit about CRF and already understand a little bit what it is about, I will try the conversion as a learning exercise based on the CRF quality parameter.
